### PR TITLE
Added Wiki pages. Updated README with links to newly created wiki pages.

### PR DESCRIPTION
**Pull request by ragv**
1. Added [Wiki pages](https://github.com/BinPy/BinPy/wiki).
   
   Check [this](https://github.com/BinPy/BinPy/wiki/_pages) for a complete list of all the newly created wiki pages [ Some are incomplete ].
   
   Please edit incomplete / incorrect entries in those newly created wiki pages.
2. Updated README.md with wiki links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,6 +186,8 @@ __DEV NOTE:__
    
    To learn more about the `autopep8` tool visit [here](https://www.google.co.in/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0CCkQFjAA&url=https%3A%2F%2Fpypi.python.org%2Fpypi%2Fautopep8%2F&ei=SjFIU7jkIcWKrQfE5oDgBQ&usg=AFQjCNGP0o38e1Ia6S7_TfsDIJrvgdGAug&sig2=Yp4VZe9UepdYtoCF_mcBFg).
 
+ - Ensure that all the tests pass by running `nosetests && nosetests3` in `BinPy\BinPy\tests` directory.
```
Comment (debugger22) [inline README.md]: '&&' ?

Comment (ragv) [inline README.md]: shell command to run both python 3 and python 2 tests one after another !

Comment (debugger22) [inline README.md]: In this case second one will run only if first one succeeds.

Comment (debugger22) [inline README.md]: I would prefer something like this `cmd1 ; cmd2`

Comment (ragv) [inline README.md]: okay will edit it that way !!

**Comment by debugger22** [comment]
Thanks!

**Pull request merged**